packages/manager/modules/pci/src/projects/project/instances/instances.service.js

```diff
@@ -305,16 +305,22 @@ export default class PciProjectInstanceService {
   }
 
   getPrivateNetworks(projectId) {
-    return this.OvhApiCloudProjectNetwork.Private()
-      .v6()
-      .query({
-        serviceName: projectId,
-      })
-      .$promise.then((networks) =>
-        filter(networks, {
-          type: 'private',
-        }),
-      );
+    return this.$http
+      .get(`/cloud/project/${projectId}/network/private`)
+      .then(({ data }) => data);
+  }
+
+  getSubnets(projectId, privateNetwork) {
+    return this.$http
+      .get(
+        `/cloud/project/${projectId}/network/private/${privateNetwork.id}/subnet`,
+      )
+      .then(({ data }) => {
+        return {
+          ...privateNetwork,
+          subnet: [...data],
+        };
+      });
```

Comment:
I wonder if this should return the subnet and aggregation should be done where it's needed